tests/integration/test_charm.py

```diff
@@ -212,13 +216,12 @@ async def test_setup_discourse(
     # Extract the CSRF token
     parsed_admin: BeautifulSoup = BeautifulSoup(response.content, features="html.parser")
     assert parsed_admin.head
-    csrf_token = parsed_admin.head.find("meta", attrs={"name": "csrf-token"}).get("content")  # type: ignore
+    csrf_token = parsed_admin.head.find("meta", attrs={"name": "csrf-token"}).get(  # type: ignore
+        "content"
+    )
 
     logger.info("Getting admin API key")
 
-    # Finally create an API Key, which will be used on the next integration tests
```

Comment:
We should probably keep the comment

Reply:
We sure do, I don't know how it got deleted, I'm putting it back. Thanks!